File: study08/backend/next/company/backend/models/Product.js
const { DataTypes } = require('sequelize');
const sequelize = require('../config/database');

const Product = sequelize.define('Product', {
    pno: { type: DataTypes.INTEGER, autoIncrement: true, primaryKey: true },
    cate: { type: DataTypes.STRING },
    pname: { type: DataTypes.STRING },
    pcontent: { type: DataTypes.TEXT },
    img1: { type: DataTypes.STRING },
    img2: { type: DataTypes.STRING },
    img3: { type: DataTypes.STRING },
    resdate: { type: DataTypes.DATE, defaultValue: DataTypes.NOW },
    hits: { type: DataTypes.INTEGER, defaultValue: 0 }
}, {
    tableName: 'product',
    timestamps: false
});

module.exports = Product;

File: study08/backend/springframework5/company/src/main/java/com/company/domain/Product.java
package com.company.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Product {
    private int pno;
    private String cate;
    private String pname;
    private String pcontent;
    private String img1;
    private String img2;
    private String img3;
    private String resdate; 
    private int hits;
}
